proto/internalapi/sensor/compliance_nodescan.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option go_package = "sensor";
+
+import "google/protobuf/timestamp.proto";
+import "scanner/api/v1/note.proto";
+import "scanner/api/v1/component.proto";
+
+
+import "gogoproto/gogo.proto";
+
+package sensor;
+
+// Names and data types are designed to follow proto/storage/node.proto as close as possible
+// Next tag: 7
+message FullNodeScan {
```

Comment:
Mentioned on Slack, but I wonder if something like `NodeScanV2` would also work? Makes it clear this is the newer version of node scanning